### ISSUE TITLE
Better results on map

### ISSUE THREAD

**Issue by ManChrys** [Livox-SDK/livox_mapping#32]
Hello guys,

i created a pcd file after using livox_mapping but the results are very poor, something that doesnt happen in rosbag pointcloud.

![Screenshot from 2021-07-26 15-21-12](https://user-images.githubusercontent.com/67521662/126987753-ab3895ad-c122-4cf6-8acf-a0a786f533e1.png)

**Comment by Livox-SDK** [comment]
Hi ManChrys,

Do you mean that when you viewing the mapping result simultaneously the result showing OK, However, after you saving the result as PCD file the results trends poor?

**Comment by ManChrys** [comment]
> Do you mean that when you viewing the mapping result simultaneously the result showing OK, However, after you saving the result as PCD file the results trends poor?

Exactly!!!

I tried livox mapping pkg to mapping an area and i saved the pcd. After that I tried to import the pcd on CloudCompare but the result is bad as you saw above . When I tried to mapping the same area with livox viewer(convert lvx to rosbag) the bag file on rviz it was very nice. Is there any way to make the point cloud from pcd better or to convert lvx to another common pointcloud format like las,laz, ply?
